util/internal/integrationtest/backend/elasticsearch/src/main/java/org/hibernate/search/util/impl/integrationtest/backend/elasticsearch/ElasticsearchTestHostConnectionConfiguration.java

```diff
@@ -53,20 +54,20 @@ private ElasticsearchTestHostConnectionConfiguration() {
 	}
 
 	public boolean isAws() {
-		return awsSigningEnabled;
+		return awsSigningEnabled != null && awsSigningEnabled;
 	}
 
 	public void addToBackendProperties(Map<String, ? super String> properties) {
 		properties.put( "hosts", hosts );
 		properties.put( "protocol", protocol );
 		properties.put( "username", username );
 		properties.put( "password", password );
-		properties.put( "aws.signing.enabled", String.valueOf( awsSigningEnabled ) );
+		properties.put( "aws.signing.enabled", awsSigningEnabled == null ? null : awsSigningEnabled.toString() );
 		properties.put( "aws.region", awsRegion );
 		properties.put( "aws.credentials.type", awsCredentialsType );
 		properties.put( "aws.credentials.access_key_id", awsCredentialsAccessKeyId );
 		properties.put( "aws.credentials.secret_access_key", awsCredentialsSecretAccessKey );
-		if ( awsSigningEnabled ) {
+		if ( awsSigningEnabled != null && awsSigningEnabled ) {
```

Comment:
Maybe you can reuse `isAws()`. It looks the same.

Reply:
Right; it should actually be just `awsSigningEnabled != null`.

Reply:
Actually I was looking in the wrong place; you're absolutely right.